<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::NoDrag</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Mapbook Info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Mapbook name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="mapbookLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gridLabel">
             <property name="text">
              <string>Position x: y:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Tile properties</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QToolButton" name="floorButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="selectorLabel">
               <property name="text">
                <string>Selected tile:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="tileDeleteButton">
               <property name="text">
                <string>Delete Tile</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Wall properties</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="0">
              <widget class="QToolButton" name="wallLeftButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QToolButton" name="wallRightButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QToolButton" name="wallUpButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QToolButton" name="wallDownButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionMap_Preferences"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionText"/>
   <addaction name="actionTile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as ..</string>
   </property>
  </action>
  <action name="actionMap_Preferences">
   <property name="text">
    <string>Map Preferences</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="toolTip">
    <string>Write text</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tile</string>
   </property>
   <property name="toolTip">
    <string>Add Tiles</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
